ScreenToGif/ViewModel/ApplicationViewModel.cs

```diff
@@ -1310,13 +1310,13 @@ internal bool InstallUpdate(bool wasPromptedManually = false)
 
             //msiexec /i PATH INSTALLDIR="" INSTALLAUTOMATICALLY=yes INSTALLPORTABLE=No ADDLOCAL=Binary
             //msiexec /a PATH TARGETDIR="" INSTALLAUTOMATICALLY=yes INSTALLPORTABLE=yes ADDLOCAL=Binary
-                
+
             var startInfo = new ProcessStartInfo
             {
                 FileName = "msiexec",
                 Arguments = $" {(isInstaller ? "/i" : "/a")} \"{nonRoot}\"" +
                             $" {(isInstaller ? "INSTALLDIR" : "TARGETDIR")}=\"{AppDomain.CurrentDomain.BaseDirectory}\" INSTALLAUTOMATICALLY=yes INSTALLPORTABLE={(isInstaller ? "no" : "yes")}" +
-                            $" ADDLOCAL=Binary{(isInstaller ? ",Auxiliar" : "")}{(hasGifski ? ",Gifski" : "")}" +
+                            $" ADDLOCAL=Binary{(isInstaller ? ",Auxiliary" : "")}{(hasGifski ? ",Gifski" : "")}" +
```

Comment:
This one unfortunately cannot be changed as it's a special parameter used by the installer. :/

Reply:
@NickeManarin What is `Auxiliar`?

Is https://github.com/NickeManarin/ScreenToGif/pull/1112/files#diff-d61513607e5408947516a4b62f574b33cb5d5d00c24e6416233f0d32eaa74b66L915 subject to this too?

Reply:
baddac0

Reply:
Just that line. :)

It's a typo/mistranslation as it's written as "Auxiliar" in my native language, it should be Auxiliary.
But since it was already written as Auxiliar in the installer, it's expecting that word.
I'll have to add some special treatment in the installer to handle the correct flag name.

Reply:
@NickeManarin Noted!  And thank you for the explanation as well.